docs/app/Examples/views/Comment/Content/CommentExampleReplyForm.js

```diff
@@ -0,0 +1,29 @@
+import React from 'react'
+import { Button, Comment, Form, Icon } from 'stardust'
+
+const CommentExampleReplyForm = () => (
+  <Comment.Group>
+    <Comment>
+      <Comment.Avatar as='a' src='http://semantic-ui.com/images/avatar/small/steve.jpg' />
+      <Comment.Content>
+        <Comment.Author as='a'>Steve Jobes</Comment.Author>
+        <Comment.Metadata>
+          <div>2 days ago</div>
+        </Comment.Metadata>
+        <Comment.Text>Revolutionary!</Comment.Text>
+        <Comment.Actions>
+          <Comment.Action active>Reply</Comment.Action>
+        </Comment.Actions>
+        <Comment.ReplyForm>
```

Comment:
I see the `reply` class is required for the form when used in a comment or after all comments.  I think we should explore some other API options for adding the class.

This pattern reminds me of the `label` and `button` class being required for Inputs that are "labeled` or have an "action".  In the Input case the label/action is passed through props, so we have an opportunity to add the className without requiring additional components or props to add the class.

Ideas:

### Form `reply` prop

We could simply add a `reply` prop to the Form.  It would clean this up and clarify things.  However, this prop is not applicable anywhere except in comments, so it seems sub optimal.

### Comment `replyForm` prop

I think this one makes more sense since you'll want the same form in all comments.  Rather than pasting a reply form in every comment and one at the end of all comments, we could create a single comment reply form and pass it as a prop to each comment:

```jsx
const replyForm = <Form />

<Comment.Group replyForm={replyForm} />
// and/or
<Comment replyForm={replyForm} />
```

### Looping Comments

This brings me to another realization, I think the actual React usage of the Comment will be to create a single comment component, query a server for a list of comments, then loop over the comments mapping the single comment component.  Because of this, I don't think most users will ever build out a comment thread in a hard coded sense.

That would put the Form `reply` prop back on the table since there would likely only be one comment component in your project (reply form included) that you would map over your comment data.